src/scss/common/accordions.scss

```
@@ -12,6 +12,11 @@
   min-width: 130px;
   max-width: 130px;
   text-align: center;
+
+  @media (max-width: 575px) {
```

This can be done with a Bootstrap mixin that doesn't rely on hard-coding the current Bootstrap pixel value:
https://getbootstrap.com/docs/4.6/layout/overview/#responsive-breakpoints

I think:

```suggestion
  @include media-breakpoint-down(xs)
```
is exactly equivalent.